assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -220,204 +119,15 @@ const Checkout = ( { attributes, scrollToTop } ) => {
-									'woo-gutenberg-products-block'
-								) }
-							>
-								<AddressForm
-									id="billing"
-									onChange={ setBillingFields }
-									type="billing"
-									values={ billingFields }
-									fields={ Object.keys(
-										defaultAddressFields
-									) }
-									fieldConfig={ addressFieldsConfig }
-								/>
-							</FormStep>
-						) }
-						{ needsShipping && (
-							<FormStep
-								id="shipping-option"
-								disabled={ checkoutIsProcessing }
-								className="wc-block-checkout__shipping-option"
-								title={ __(
-									'Shipping options',
-									'woo-gutenberg-products-block'
-								) }
-								description={
-									getShippingRatesRateCount( shippingRates ) >
-									1
-										? __(
-												'Select shipping options below.',
-												'woo-gutenberg-products-block'
-										  )
-										: ''
-								}
-							>
-								{ isEditor &&
-								! getShippingRatesPackageCount(
-									shippingRates
-								) ? (
-									<NoShippingPlaceholder />
-								) : (
-									<ShippingRatesControl
-										noResultsMessage={ __(
-											'There are no shipping options available. Please ensure that your address has been entered correctly, or contact us if you need any help.',
-											'woo-gutenberg-products-block'
-										) }
-										renderOption={
-											renderShippingRatesControlOption
-										}
-										shippingRates={ shippingRates }
-										shippingRatesLoading={
-											shippingRatesLoading
-										}
-									/>
-								) }
-							</FormStep>
-						) }
-						{ cartNeedsPayment && (
-							<FormStep
-								id="payment-method"
-								disabled={ checkoutIsProcessing }
-								className="wc-block-checkout__payment-method"
-								title={ __(
-									'Payment method',
-									'woo-gutenberg-products-block'
-								) }
-								description={
-									Object.keys( paymentMethods ).length > 1
-										? __(
-												'Select a payment method below.',
-												'woo-gutenberg-products-block'
-										  )
-										: ''
-								}
-							>
-								<StoreNoticesProvider context="wc/payment-area">
-									<PaymentMethods />
-								</StoreNoticesProvider>
-							</FormStep>
-						) }
-						{ attributes.showOrderNotes && (
-							<FormStep id="order-notes" showStepNumber={ false }>
-								<CheckoutOrderNotes
-									disabled={ checkoutIsProcessing }
-									onChange={ setOrderNotes }
-									placeholder={
-										needsShipping
-											? __(
-													'Notes about your order, e.g. special notes for delivery.',
-													'woo-gutenberg-products-block'
-											  )
-											: __(
-													'Notes about your order.',
-													'woo-gutenberg-products-block'
-											  )
-									}
-									value={ orderNotes }
-								/>
-							</FormStep>
-						) }
-					</CheckoutForm>
+					<CheckoutForm
+						loginToCheckoutUrl={ loginToCheckoutUrl }
```

Comment:
Do we need to pass the login URL through all the layers? Could be local to the component that needs it, or refactored out to utils/settings.

Reply:
Ah, good point. Moved it to a `utils.js` file in bc03b8906bcb8990600a195e7cce1c98997f24b3.